manifests/script-lab-react-localhost.outlook.xml

```diff
@@ -727,6 +727,9 @@
         </bt:String>
       </bt:LongStrings>
     </Resources>
+         <ExtendedPermissions>
```

Comment:
Would we need to bump up the version number again? I believe not since the previous bumped up version didn't make it through but just wanted to make sure

Reply:
Since we didn't publish the latest changes I made recently, I don't think we need to bump the version again

Reply:
that's right, no need to bump this time